README.md

```diff
@@ -0,0 +1,49 @@
+# Google Code-in Leaders
+***
+This is a website to show the current leaders from all organizations in GCI.
+***
+ * [Installation](#installation)
+ * [Languages](#languages)
+ * [Usage](#usage)
+ * [Getting Involved](#getting-involved)
+ * [Support](#support)
+ * [Authors](#authors)
+ * [License](#license)
+***
+### Installation
+```
+npm install
+```
+***
+### Languages:
+* Node.js
+* JSON
+***
+### Usage:
+```
+npm run build
+```
+***
+### Getting Involved
+If you would like to be a part of the coala community, you can check out our [Getting In Touch](http://coala.readthedocs.io/en/latest/Help/Getting_In_Touch.html) page or ask us at our active Gitter channel, where we have maintainers from all over the world. We appreciate any help!
+
+We also have a [newcomer guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html) to help you get started by fixing an issue yourself! If you get stuck anywhere or need some help, feel free to contact us on Gitter or drop a mail at our [newcomer mailing list](https://groups.google.com/forum/#!forum/coala-newcomers).
+***
+### Support
+Feel free to contact us at our [Gitter channel](https://gitter.im/coala/coala), we'd be happy to help!
+
+If you are interested in commercial support, please contact us on the Gitter channel as well.
+
+You can also drop an email at our [mailing list](https://github.com/coala/coala/wiki/Mailing-Lists).
+***
+### Authors
+You can contact to author for more information.
+[blazeu](mailto:neversleepman@tfwno.gf)
+***
+### License
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
We still haven't added a License to this repo yet.

Reply:
And it can be autodetected using shields.io, so may as well do that instead of hardcoding it.

Reply:
License is MIT according to package.json

Reply:
Yeah just found a issue for it https://github.com/coala/gci-leaders/issues/6